backend/Properties/launchSettings.json

```diff
@@ -22,6 +22,7 @@
       "launchBrowser": true,
       "launchUrl": "api/values",
       "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
This is not the correct place for this environment variable. It is set in the development launch settings already.

Reply:
This was IDE generated, has been fixed now.